notebooks/211-speech-to-text/211-speech-to-text.ipynb

```diff
@@ -0,0 +1,559 @@
+    "    log_melspectrum = np.log(np.dot(mel_basis, np.power(spec, 2)) + 2 ** -24)\n",
+    "\n",
+    "    # Normalize output\n",
+    "    normalized = (log_melspectrum - log_melspectrum.mean(1)[:, None]) / (log_melspectrum.std(1)[:, None] + 1e-5)\n",
+    "\n",
+    "    # Calculate padding\n",
+    "    remainder = normalized.shape[1] % padding\n",
+    "    if remainder != 0:\n",
+    "        return np.pad(normalized, ((0, 0), (0, padding - remainder)))[None]\n",
+    "    return normalized[None]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d86046e",
+   "metadata": {},
+   "source": [
+    "### Run Conversion from Audio to Mel Format\n",
+    "\n",
+    "In this step you want to convert current audio file into [Mel scale](https://en.wikipedia.org/wiki/Mel_scale)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mel_basis, spec = audio_to_mel(audio=audio.flatten(), sampling_rate=sampling_rate)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e3276fa",
+   "metadata": {},
+   "source": [
+    "### Visualise Mel Spectogram\n",
+    "\n",
+    "If you want to know more about mel spectogram follow this [link](https://towardsdatascience.com/getting-to-know-the-mel-spectrogram-31bca3e2d9d0). First image visualise Mel frequency spectogram, second one presents filter bank for converting Hz to Mels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bef57088",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "librosa.display.specshow(data=spec, sr=sampling_rate, x_axis='time', y_axis='log');\n",
+    "plt.show();\n",
+    "librosa.display.specshow(data=mel_basis, sr=sampling_rate, x_axis='linear');\n",
+    "plt.ylabel('Mel filter');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "503f1778",
+   "metadata": {},
+   "source": [
+    "### Adjust Mel scale to Input\n",
+    "\n",
+    "Before proceeding to network, you want to be sure that input is well prepared."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a63159b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "audio = mel_to_input(mel_basis=mel_basis, spec=spec)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e4aec612",
+   "metadata": {},
+   "source": [
+    "## Load Model\n",
+    "\n",
+    "When everything is prepared, you can finally read and load network. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7d5dc5c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = IECore()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ce3fc33e",
+   "metadata": {},
+   "source": [
+    "You may choose to run the network on multiple devices by default it will load the model on the CPU (you can choose manually CPU, GPU, MYRIAD, etc.) or let the engine choose the best available device (AUTO).\n",
```

Comment:
```suggestion
    "You may choose to run the network on multiple devices. By default, it will load the model on the CPU (you can choose manually CPU, GPU, MYRIAD, etc.) or let the engine choose the best available device (AUTO).\n",
```